src/Ledger/GovernanceActions.lagda

```diff
@@ -124,28 +138,12 @@ module _ (pp : PParams) (ccThreshold' : Maybe ℚ) where
     vote : ℚ → Maybe ℚ
     vote = just
 
-  threshold : GovAction → GovRole → Maybe ℚ
-  threshold = λ where
-    NoConfidence
-               → λ { CC → noVote           ; DRep → vote P1     ; SPO → vote Q1 }
-    (NewCommittee _ _ _) → case ccThreshold' of λ where
-      (just _) → λ { CC → noVote           ; DRep → vote P2a    ; SPO → vote Q2a }
-      nothing  → λ { CC → noVote           ; DRep → vote P2b    ; SPO → vote Q2b }
-    (NewConstitution _ _)
-               → λ { CC → vote ccThreshold ; DRep → vote P3     ; SPO → noVote }
-    (TriggerHF _)
-               → λ { CC → vote ccThreshold ; DRep → vote P4     ; SPO → vote Q4 }
-    (ChangePParams x)
-               → λ { CC → vote ccThreshold ; DRep → vote (P5 x) ; SPO → noVote }
-    (TreasuryWdrl _)
-               → λ { CC → vote ccThreshold ; DRep → vote P6     ; SPO → noVote }
-    Info       → λ { CC → vote 2ℚ          ; DRep → vote 2ℚ     ; SPO → vote 2ℚ }
-
 -- TODO: this doesn't actually depend on PParams so we could remove that argument,
 --       but we don't have a default ATM
 canVote : PParams → GovAction → GovRole → Set
 canVote pp a r = Is-just (threshold pp nothing a r)
 \end{code}
+\end{figure*}
```

Comment:
Please add a caption (do not exceed one line, there is minimal space left in this page) and `\label{fig:..}` to be consistent with the rest of the document.